scripts/generate-release-notes.js

```diff
@@ -80,13 +84,14 @@ async function finalizeReleaseNotes (tag, app) {
 
   // Make sure that the tag passed in is valid
   let tags = await run('git', ['tag'])
-  if (!tags.includes(tag)) {
+
+  if (!tags.split('\n').includes(tag)) {
     console.log('Invalid tag. Run `git tag` to see available tags.')
     process.exit(1)
   }
 
   try {
-    let result = await run('git', ['log', `${tag}...HEAD`, `--pretty=format:commit:%H%n%B${delimiter}`])
+    let result = await run('git', ['log', `${tag}...HEAD`, `--pretty=format:commit:%H%n%B${delimiter}`, '--'])
```

Comment:
The `--` will tell the log command that the `${tag}...HEAD` are references and not file paths as file paths would come after the `--`